jupyterlab/labapp.py

```diff
@@ -538,7 +538,8 @@ class LabApp(NBClassicConfigShimMixin, LabServerApp):
     )
     flags['expose-app-in-browser'] = (
         {'LabApp': {'expose_app_in_browser': True}},
-        "Expose the global app instance to browser via window.jupyterlab."
+        """Expose the global app instance to browser via window.jupyterapp.
+        It is also available via the deprecated window.jupyterlab name."""
```

Comment:
Also if we start deprecating `window.jupyterlab` for 3.2.x, maybe we could actually drop it for 4.0